scripts/envsetup.sh

```diff
@@ -35,23 +39,26 @@ fi
 METADIR=${METADIR:-${SOURCEDIR}/../..}
 
 if [[ ! -f "${BUILDDIR}/conf/local.conf" ]]; then
-  declare -a DISTRO_CONFIGS=${supported_distros[$DISTRO]}
-  [[ -n ${DISTRO_CONFIGS[@]} ]] && { echo "Using (${DISTRO_CONFIGS[*]}) for the specified distro '$DISTRO'"; } || { echo "The specified distro $DISTRO is not supported"; return 1; }
+  if [ -z "${supported_distros[$DISTRO]}" ]; then
+    echo "The specific distro $DISTRO is not supported"
```

Comment:
```suggestion
    echo "The specified distro $DISTRO is not supported"
```
Sorry for being pedantic. Maybe worth printing the Usage string here as well?